src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/AddressDto.java

```diff
@@ -7,44 +7,55 @@
 
 public class AddressDto {
 
+    public static final String PROPERTY_NAME_NUMBER_FIELD = "property_name_number";
```

Comment:
Would remove the _FIELD off the end of this constant name, it's not there on the others.

Reply:
Oh, I see it is there in other DTOS. Maybe it needs adding on the others, or maybe these values are already used elsewhere (trusts?) and we should be careful about what we change... Need to be consistent with this naming, however. Requires some investigation.

Reply:
Updated names with FIELD suffix.